pkg/workloads/cortex/lib/util.py

```diff
@@ -297,3 +297,7 @@ def render_jinja_template(jinja_template_file: str, context: dict) -> str:
 
     template = env.get_template(str(template_path.name))
     return template.render(**context)
+
+
+def available_aws_credentials() -> bool:
```

Comment:
nit: consider renaming to `are_aws_credentials_available()`

Reply:
Removed function.